src/discussions/posts/post/Post.jsx

```diff
@@ -35,6 +35,7 @@ function Post({
   const { enableInContextSidebar } = useContext(DiscussionContext);
   const courseId = useSelector((state) => state.config.id);
   const topic = useSelector(selectTopic(post.topicId));
+  const CourseWideTopicURL = `${window.location.origin}/${courseId}/topics/${post.topicId}`;
```

Comment:
instead of using window.location.origin please use the getApiBaseUrl from constants.js.